packages/website/components/Sidebar.tsx

```diff
@@ -35,6 +39,50 @@ interface Props {
   currentPage?: string;
 }
 
+const components: Array<SidebarSectionType | SidebarLinkType> = [
+  {
+    type: 'section',
+    links: sidebarLinks.layoutComponents,
+    title: 'Layout Components',
```

Comment:
I am not convicted that all the sections that have more than one component should be on the top of the `components` section, but we can reshuffle later if we decide otherwise.